.travis.yml

```diff
@@ -13,11 +13,38 @@ install:
   - pip install -r requirements-travis.txt
   - echo 'SECRET_KEY = "evap-travis-secret-key"' >> evap/localsettings.py
 
-env:
-  - COMMAND="pylint evap -j 0"
-  - COMMAND="deployment/test_backup_and_restore_on_travis.sh"
-  - COMMAND="coverage run manage.py test && codecov -X gcov"
-  - COMMAND="python manage.py test --debug-mode"
-  - COMMAND="python manage.py test --reverse"
-script:
-  - eval $COMMAND
+jobs:
+  include:
+    - name: Lint Python
+      script:
+        - pylint evap -j 0
+    - name: Test Backup Script
+      script:
+        - deployment/test_backup_and_restore_on_travis.sh
+    - name: Test Python
+      script:
+        - coverage run manage.py test && codecov -X gcov
+    - name: Test Python (debug mode)
+      script:
+        - python manage.py test --debug-mode
+    - name: Test Python (backwards)
+      script:
+        - python manage.py test --reverse
+    - name: Test Typescript
+      services:
+        - postgresql
+        - redis
+        - xvfb
+      install:
+        - pip install -r requirements-travis.txt
+        - echo 'SECRET_KEY = "evap-travis-secret-key"' >> evap/localsettings.py
+        - wget https://raw.githubusercontent.com/nvm-sh/nvm/v0.36.0/install.sh --no-verbose --output-document - | bash
+        - deployment/install_dart_sass.sh
+        - . "${HOME}/.nvm/nvm.sh"
+        - nvm install 15.4.0
```

Comment:
We have this version string written out in several places, and we will probably have to change it quite often (new node releases happen like twice per year).

I see that it is hard to unify these locations and only put the version in one place. Have you considered using "stable" at the risk of stuff failing (which should be caught by travis)? Otherwise, do we want to document these locations?

Reply:
I think just using `stable` should suffice, made some things easier to do :)